crates/common/src/lib.rs

```diff
@@ -175,6 +177,12 @@ pub struct ResourcePricePerUnit(pub u128);
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Deserialize, Serialize, Default, Dummy)]
 pub struct TransactionVersion(pub Felt);
 
+impl GasPrice {
+    pub fn as_hex(&self) -> String {
+        format!("0x{:X}", self.0)
+    }
```

Comment:
Is this used anywhere?

Reply:
No; was used as part of the previous commit state.

Was trying this out as an alternative to serde_conv. e.g. create a `trait ToHexString` and implement it for all felt and integer newtypes. This would then be useful for manual serde DTO implementations. But not needed.